integration/framework.go

```diff
@@ -457,3 +459,36 @@ func (f *framework) ApplyAWSConfigPatch(patch []PatchSpec, clusterName string) e
 
 	return nil
 }
+
+func GetVersionBundleVersion(bundle []versionbundle.Bundle, vType string) (string, error) {
+	validVTypes := []string{"", "current", "wip"}
+	var isValid bool
+	for _, v := range validVTypes {
+		if v == vType {
+			isValid = true
+			break
+		}
+	}
+	if !isValid {
+		return "", fmt.Errorf("%q is not a valid version bundle version type", vType)
+	}
+
+	var output string
+	log.Printf("Tested version %q", vType)
+
+	// sort bundle by time to get the newest vbv.
+	s := versionbundle.SortBundlesByTime(bundle)
+	sort.Sort(s)
+	for i := len(s) - 1; i >= 0; i-- {
```

Comment:
I think using reverse is a bit cleaner https://golang.org/pkg/sort/#Reverse

Reply:
Yep much better :) done